Tabloid/client/src/components/TagManagment.js

```diff
@@ -0,0 +1,16 @@
+const baseUrl = 'api/Tag';
```

Comment:
This file does not a component. This file should be in the modules directory.

Reply:
@heymonicakay Hey I changed the tag management and move it to modules